src/views/count.vue

```diff
@@ -193,19 +240,57 @@
           console.error(err);
         }
         return resp;
-      }
+      },
+      updateVarianceReason(event: any) {
+        this.product.reason = event.detail.value
+      },
+      async presentAlertOnVarianceUpdate() {
+        if (this.product.variance && this.product.reason) {
+          const alert = await alertController.create({
+            header: this.$t("Log variance"),
+            message: this.$t("Are you sure you want to update the inventory variance?"),
```

Comment:
We should add another line to it like The inventory will be immediately updated and cannot be undone.